Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackFindingAlgorithm.hpp

```diff
@@ -42,6 +43,7 @@ class TrackFindingAlgorithm final : public BareAlgorithm {
       Options::BFieldVariant magneticField);
 
   struct Config {
+    std::string inputMeasurements;
```

Comment:
Could you add some comment for the `inputMeasurements`?

Reply:
Added some comments.